src/Context/AnnotatedClassDiscovery.php

```diff
@@ -11,7 +11,13 @@
 use \Drupal\Core\Plugin\Discovery\AnnotatedClassDiscovery as CoreAnnotatedClassDiscovery;
 
 /**
- * Extends the annotation class discovery for usage with Rules context.
```

Comment:
This was right actually, it's more than just annotations - let's leave the old comment.